README.md

```diff
@@ -87,7 +87,16 @@ hipsolverSgetrf(hipsolverHandle_t handle,
                 int*              devInfo);
 ```
 
-## Supported Fuctionality
+## Special Considerations with the rocSOLVER Backend
+Due to differences in implementation and API design between rocSOLVER and cuSOLVER, there are special considerations that should be taken into account when using hipSOLVER with the rocSOLVER backend.
```

Comment:
I disagree. The rocSOLVER backend is, in a sense, jury-rigged to fit into the cuSOLVER API design, and this causes issues that the cuSOLVER backend does not have. I don't think it makes sense to frame the ignored info arguments or flip-flopping between memory models as differences between the backends - they are unfortunate side effects caused by mapping one library design into another.

I'll see if I can play around with the wording a little bit, but I think it would be best to keep the same framing.